### ISSUE TITLE
AU plugin version?

### ISSUE THREAD

**Issue by trevormeier** [falkTX/Carla#1553]
Hi there—are there any plans to provide an AU format plugin for hosting Carla for e.g. in Logic and other AU hosts?

**Comment by falkTX** [comment]
No, but there might be an integrated workaround late next year.
